requirements.txt

```diff
@@ -10,3 +10,6 @@ phonenumbers==7.2.6
 django-cities-light==3.2.0
 Unidecode==0.4.19
 django-braces==1.9.0
+mccabe==0.6.1,
+pycodestyle==2.3.1,
```

Comment:
say flake8 and specify its version here. Also include both #480 and #481 in a single pull request. In .travis.yml run flake8 on vms.